sources/shopify_dlt/README.md

```diff
@@ -0,0 +1,60 @@
+# Shopify
+
+Shopify is an easy-to-use e-commerce solution that allows anyone to create and manage their own online store. This `dlt` verified source is designed to efficiently load data from multiple endpoints that are:
+
+| Endpoint | Description |
+| --- | --- |
+| customers | individuals or entities who have created accounts on a Shopify-powered online store |
+| orders  | transactions made by customers on an online store |
+| products | the individual items or goods that are available for sale |
+
+## Initialize the pipeline with Hubspot verified source
```

Comment:
you have Hubspot in title instead of Shopify, I would recommend keep only "Initialize the pipeline" for every readme. it reduces the chance of making mistakes

Reply:
done